python/pyspark/rdd.py

```diff
@@ -188,6 +188,12 @@ def __init__(self, jrdd, ctx, jrdd_deserializer=AutoBatchedSerializer(PickleSeri
         self._id = jrdd.id()
         self.partitioner = None
 
+    def __enter__(self):
```

Comment:
Thats an interesting approach @MechCoder I think that could be a way to clarify how to expect to use the context manager to users.